test/e2e/multi-cluster-deployment/README.md

```diff
@@ -15,12 +15,10 @@ cd test/e2e/multi-cluster-deployment
 ./run-test.sh
 ```
 
-## Running the test in OCP cluster
+## Running the test in three OCP clusters
```

Comment:
Not sure "three" is needed. We don't mention this explicitly for KIND or any other env. 
"Running the test using OCP clusters"

Reply:
I pushed a revision that makes the structure of the two sub-headings parallel. I think it is important to be very clear about the infrastructure aspect.